marimbabot_vision/scripts/generate_augmented_data.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import os
```

Comment:
It looks like the script is added in this PR, but it is present in the main branch. [Ref](https://github.com/UHHRobotics22-23/robot_project/blob/0ecf498450e0dc54437aa5908a40cc08132382cc/marimbabot_vision/scripts/generate_augmented_data.py)

You might want to pull and rebase this with the main branch.

Reply:
I already did